src/data/mainnet/tokens-info.json

```
@@ -373,5 +373,12 @@
     "decimals": 18,
     "name": "Wrapped Ether",
     "symbol": "WETH"
+  },
+  {
+    "address": "0x34C11A932853Ae24E845Ad4B633E3cEf91afE583",
```

```suggestion
    "address": "0x34c11a932853ae24e845ad4b633e3cef91afe583",
```
this should fix your failing test